proctor-common/src/main/java/com/indeed/proctor/common/Proctor.java

```diff
@@ -515,4 +522,54 @@ private boolean isIncognitoEnabled(@Nonnull final Map<String, Object> inputConte
                 .map(value -> value.equalsIgnoreCase("true"))
                 .orElse(false);
     }
+
+    private void populateProperties(
```

Comment:
Could you add a comment for this method? Or maybe just a short description here?

Reply:
Added